src/app/desktop/window/window-frame.component.ts

```diff
@@ -57,6 +57,7 @@ export class WindowFrameComponent implements OnInit {
       this.dragging = true;
       this.dragStartPos = [event.clientX, event.clientY];
       this.dragStartWindowPos = [this.delegate.position.x, this.delegate.position.y];
+      document.getElementsByClassName("window-header")[0].classList.add("dragging");
```

Comment:
Why don't you specify the class in the template using HTML. Also if you directly manipulate HTML using JS use the Angular Methods.

Reply:
?

Reply:
Add
```
[class.dragging]="dragging"
```
to the HTML element in the template.